packages/orbit-components/src/Stepper/StepperStateless/index.tsx

```diff
@@ -103,6 +105,8 @@ const StepperStateless = ({
         }}
         onBlur={onBlur}
         onFocus={onFocus}
+        aria-label={ariaLabel}
+        aria-live={ariaLive}
```

Comment:
tbh, I am not sure if it's the best idea to include `aria-level` attribute, but I included it as the devs might want to set up this attribute to ensure the user that the value in the input component was de/increased on key press (?). WDYT?